packages/backend/src/database/migrations/20230201163036_add_scheduler_tables.ts

```diff
@@ -0,0 +1,76 @@
+import { Knex } from 'knex';
+
+const SchedulerTableName = 'scheduler';
+const SchedulerSlackTargetTableName = 'scheduler_slack_target';
+export async function up(knex: Knex): Promise<void> {
```

Comment:
We should probably run `workerUtils.migrate()` here  too

https://github.com/graphile/worker#makeworkerutilsoptions-workerutilsoptions-promiseworkerutils

Reply:
That should be done when the server starts, to guarantee that the database matches the installed version of the library.